dmapiclient/base.py

```diff
@@ -42,11 +60,39 @@ def iter_method(self, *args, **kwargs):
     return iter_method
 
 
+class GenericRequester(object):
+    def __init__(self, client, prefix='/'):
+        self.client = client
+        self.prefix = prefix
+
+    def _requester(self, *args):
+        tokens = ['/'] + list(str(_) for _ in args)
+        path = os.path.join(*tokens)
+        path = path.lstrip(os.sep)
+        prefix = os.path.join(self.prefix, path)
+        return GenericRequester(self.client, prefix=prefix)
+
+    def __getattr__(self, name):
```

Comment:
excuse my ignorance, but when req.x() is called this is when __getattr__ is called?